src/js/pyodide.ts

```diff
@@ -99,6 +109,14 @@ export async function loadPyodide(
      * Default: ```${indexURL}/pyodide-lock.json```
      */
     lockFileURL?: string;
+    /**
+     * TODO(now) fill this in
+     */
+    lockFileContents?: Promise<Lockfile | string>;
+    /**
+     * TODO(now) fill this in
+     */
+    packageBaseUrl?: string;
```

Comment:
Maybe `lockFileBaseURL` as it only affects loading packages from the lockfile?

... but it might be confusing to have both `lockFileURL` and `lockFileBaseURL`.

Reply:
BTW, I think the parameter should be `URL` with all uppercase for consistency.